server/sonar-web/src/main/js/apps/projects/components/ProjectCard.js

```diff
@@ -99,6 +89,31 @@ export default function ProjectCard({ measures, organization, project, type }: P
           <Link to={{ pathname: '/dashboard', query: { id: project.key } }}>{project.name}</Link>
         </h2>
         {displayQualityGate && <ProjectCardQualityGate status={measures['alert_status']} />}
+        <div className="pull-right text-right">
+          {isPrivate && <PrivateBadge className="spacer-left" tooltipPlacement="left" />}
+          {hasTags && <TagsList tags={project.tags} customClass="spacer-left" />}
+        </div>
+        {isLeakView &&
+          isProjectAnalyzed &&
+          <div
+            className={classNames('project-card-dates note text-right pull-right', {
+              'width-100': isPrivate || hasTags
+            })}>
+            {hasLeakPeriodStart &&
+              <span>
+                {translateWithParameters(
```

Comment:
there is a little inconsistency between two messages: 

> Leak Period started 7 months ago

> Last analysis: April 26, 2017 11:38 AM

Note that the second one has a colon, but the first one doesn't. This is a bit annoying.